bundle.js

```diff
@@ -0,0 +1,167 @@
+(() => {
```

Comment:
I believe is good practice to .gitignore this file to avoid conflicts if working in a team. This file will anyway be autogenerated when running 'npm run build' locally